### PR TITLE
docs(i18n-id): translate community.ts and documentation.ts to Indonesian

### PR DESCRIPTION
**Pull request by jefrydco**
This is first attempt to translate TypeScript documentation to Indonesian. Please kindly help review the transliteration Mas @mazipan, @mandaputtra, @isatrio, @nusendra, @resir014, @grikomsn, @rizafahmi

### CODE REVIEW

**Review by nusendra — APPROVED** [review]

File: packages/typescriptlang-org/src/copy/id/community.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+export const comCopy = {
+  com_layout_title: "Bagaimana cara mempersiapkan TypeScript",
+  com_layout_description:
+    "Terhubung dengan pengguna TypeScript secara daring dan luring.",
+  com_headline: "Terhubung dengan kami",
+  com_connect_online: "Daring",
+  com_connect_online_description:
+    " Beri tahu kami apa yang bekerja dengan baik, apa yang ingin Kamu tambahkan atau tingkatkan, dan cari tahu mengenai pembaruan terbaru.",
```
Comment (mazipan) [inline packages/typescriptlang-org/src/copy/id/community.ts]: "kamu" berbeda dengan "Anda". Tidak perlu menggunakan karakter kapital di depannya

**Review by mazipan — CHANGES_REQUESTED** [review]

File: packages/typescriptlang-org/src/copy/id/community.ts (hunk 1 above)
Comment (jefrydco) [reply packages/typescriptlang-org/src/copy/id/community.ts]: Maksudnya semua kata "kamu" diganti menjadi "anda" gitu Mas?

---

## Update

Ah maaf, baru ngeh. Oke Mas, terima kasih

Comment (nusendra) [reply packages/typescriptlang-org/src/copy/id/community.ts]: kamu tanpa kapital maksudnya mas

**Review by mazipan — APPROVED** [review]

**Comment by mandaputtra** [comment]
LGTM

**Review by mandaputtra — APPROVED** [review]

**Review by resir014 — APPROVED** [review]

**Comment by jefrydco** [comment]
Since we already have several approval, can we merge this PR @orta?

**Review by grikomsn — COMMENTED** [review]
LGTM!

**Pull request merged**